azurerm/internal/services/netapp/tests/resource_arm_netapp_account_test.go

```diff
@@ -234,6 +239,10 @@ resource "azurerm_netapp_account" "test" {
     domain              = "westcentralus.com"
     organizational_unit = "OU=FirstLevel"
   }
+
+  tags = {
+    "foo" = "bar"
```

Comment:
Could we ensure capitals work
```suggestion
    "FoO" = "BaR"
```